### DISCUSSION TITLE
登陆失败

### DISCUSSION THREAD

**Discussion by fivenoodle** [xiaoyaocz/flutter_dmzj#159] [General]
提示"请求失败，请检查网络"
开了数据还是一样
版本号是2.0.6

**Comment by Schezuk** [comment]
`user.idmzj.com`和`i.idmzj.com` 昨晚出现了全国性的dns解析错误
暂时可以通过在hosts中添加`180.184.93.160 user.idmzj.com`来解决。

不过今天早上已经正常了，各地dns解析都指向指向`180.184.92.164`

**Comment by Schezuk** [comment]
`user.idmzj.com`和`i.idmzj.com` 昨晚出现了全国性的dns解析错误
暂时可以通过在hosts中添加`180.184.93.160 user.idmzj.com`来解决。

不过今天早上已经正常了，各地dns解析都指向指向`180.184.92.164`
